kuzzleio/kuzzle-containers
# Fix plugin dev and kuzzle images

## What does this PR do ?

This PR adapt the `kuzzleio/plugin-dev` and `kuzzleio/kuzzle` images to be build manually in travis.   

The `kuzzleio/plugin-dev` image will download Kuzzle source code from Github. You can specify the branch with a build argument (eg: `--build-arg kuzzle_branch=1.x`)

The `kuzzleio/kuzzle` image is based on the `kuzzleio/plugin-dev` image with a multi stage build. You can specify the tag of the `kuzzleio/plugin-dev` image that should be used to build `kuzzleio/kuzzle` (eg: `--build-arg current_tag=develop`)


### How should this be manually tested?

  - Step 1 : Build `kuzzleio/plugin-dev` with `1.x` branch : `docker build -t kuzzleio/plugin-dev:develop -f plugin-dev/Dockerfile --build-arg kuzzle_branch=1.x plugin-dev`
  - Step 2 : Build `kuzzleio/kuzzle` with `kuzzleio/plugin-dev` on tag `develop`: `docker build -t kuzzleio/kuzzle:develop -f kuzzle/Dockerfile --build-arg current_tag=develop kuzzle`
  - Step 3 :  Each image should contain the same Kuzzle version as on the `1.x` branch.
